flopy/mf6/data/dfn/gwf-evta.dfn

```diff
@@ -157,6 +157,7 @@ name ievt
 type integer
 shape (ncol*nrow; ncpl)
 reader readarray
+numeric_index true
```

Comment:
@spaulins-usgs, this is not the authoritative place to make this change.  Have you made this change in the mf6 repo also?

Reply:
This change is made in my local copy of the mf6 repo.  I will start a mf6 repo PR now.